destination.go

```diff
@@ -35,19 +38,58 @@ func (d *Destination) Config() sdk.DestinationConfig {
 	return &d.config
 }
 
-func (d *Destination) Open(_ context.Context) error {
+func (d *Destination) Open(ctx context.Context) error {
 	// Open is called after Configure to signal the plugin it can prepare to
 	// start writing records. If needed, the plugin should open connections in
 	// this function.
+	creds, err := d.config.BuildCredentials()
+	if err != nil {
+		return fmt.Errorf("failed to build credentials: %w", err)
+	}
+
+	driveService, err := drive.NewService(ctx, option.WithCredentialsJSON(creds))
+	if err != nil {
+		return fmt.Errorf("failed to open connection: %w", err)
+	}
+	d.service = driveService
+
 	return nil
 }
 
-func (d *Destination) Write(_ context.Context, _ []opencdc.Record) (int, error) {
-	// Write writes len(r) records from r to the destination right away without
-	// caching. It should return the number of records written from r
-	// (0 <= n <= len(r)) and any error encountered that caused the write to
-	// stop early. Write must return a non-nil error if it returns n < len(r).
-	return 0, nil
+func (d *Destination) Write(ctx context.Context, r []opencdc.Record) (int, error) {
+	// Log the number of records
+	sdk.Logger(ctx).Info().Int("records", len(r)).Msg("Starting file uploads...")
```

Comment:
this log will happen each time `Write` is called, so it's better to use a lower level log:
```suggestion
	sdk.Logger(ctx).Trace().Int("records", len(r)).Msg("Starting file uploads...")
```